Gopkg.toml

```diff
@@ -37,10 +37,6 @@
   name = "github.com/coreos/go-iptables"
   version = "0.3.0"
 
-[[constraint]]
-  name = "github.com/fortytw2/leaktest"
```

Comment:
Is this removed because we no longer use it?

Maybe I'll create a separate issue to make sure we add leaktest back to all tests- it picked up a few things in the early days.